Frontend/ui-driver/src/Screens/SubscriptionScreen/View.purs

```diff
@@ -76,6 +80,7 @@ screen initialState globalState =
   , name: "SubscriptionScreen"
   , globalEvents: [(\push -> do
       void $ launchAff $ flowRunner defaultGlobalState $ loadData push LoadPlans LoadAlternatePlans LoadMyPlans LoadHelpCentre ShowError initialState globalState
+      void $ launchAff $ flowRunner defaultGlobalState $ void $ runExceptT $ runBackT $ getUpiApps
```

Comment:
Do we want to call getUpiApps every time, not a good approach can be called before coming to the Subscription screen

Reply:
We need to update every when driver comes to subscription screen.